plugin/trino-iceberg/pom.xml

```diff
@@ -26,7 +26,7 @@
         <air.test.parallel>instances</air.test.parallel>
         <dep.keycloak.version>21.1.2</dep.keycloak.version>
         <!-- Nessie version (matching to Iceberg release) must be bumped along with Iceberg version bump to avoid compatibility issues -->
-        <dep.nessie.version>0.71.0</dep.nessie.version>
```

Comment:
Separate commit -> relates probably to iceberg 1.5.x bump (first commit in the PR)

Reply:
Will do, 1.5.0 release should be sometime soon (hopefully this month but of course depends). I've been using the snapshot release for development of this feature. I think what most likely would happen is when 1.5.0 is released we'd have a separate PR out for the dependency upgrade and I'd rebase this change on top of that after it gets merged. Let me know what you  think @findepi @findinpath

Reply:
So, this PR can't be merged until 1.5.0 is released right? (It has dependency on some view APIs?)

I sent a mail yesterday in Iceberg mailing list about 1.5.0 release. Lets make the release happen within this month. Then a separate PR to bump Iceberg version and this PR can be merged after that. 

But for development we can depend on snapshot version and keep the PR as WIP or Draft.

Reply:
@ajantha-bhat Yes that's what I mean. After 1.5, we'd most likely have a PR just for the upgrade and then just rebase this change.

Reply:
@findepi @findinpath Just a heads up, all the dependency updates you see in this PR such as the Iceberg 1.5 update and the Nessie update will be in a separate PR, along with any of the relevant changes that come up as part of the upgrade. https://github.com/trinodb/trino/pull/20308/files is the PR.